src/main/java/aztech/modern_industrialization/machines/init/MIMachineRecipeTypes.java

```diff
@@ -27,7 +27,6 @@
 import aztech.modern_industrialization.machines.recipe.CuttingMachineRecipeType;
```

Comment:
This file was excluded from spotless, not how it got formatted it? This should be reverted.